README.md

```diff
@@ -1,24 +1,230 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📗 Table of Contents](#-table-of-contents)
+- [📖Hesab ](#hesab-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [🚀 Project Description ](#-project-description-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [👩‍💻 Alaa Shaheen](#-alaa-shaheen)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖Hesab <a name="about-project"></a>
 
-* Database creation
+**Hesab** Is a financial management mobile app.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+> Ruby on Rails
 
-* Deployment instructions
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">RoR Views/Teplates</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.rackservers.com/">Rack</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create User**
+- **Create a group**
+- **Add expenses to a group**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Hesab](https://alaahesab.onrender.com/)
+
```

Comment:
- [ ] I think you have a problem when deploying your project I highly recommend to fix issue and make your deployed link work as expected 🤩💯 so kindly fix it since after signing up I can see this ⬇
![image](https://user-images.githubusercontent.com/58553711/221380224-f44d7d5f-6565-4070-a46b-ecd6afa76c81.png)

Reply:
Solved